### DISCUSSION TITLE
How to scroll results and preview left / right ?

### DISCUSSION THREAD

**Discussion by nowaylifer** [ibhagwan/fzf-lua#1495] [Q&A] [answered]
Up / down works, but how left / right ?

**Comment by ibhagwan** [comment, marked as answer]
Not possible with fzf ATM, you can make a feature request upstream.

If you need to see the right side you can use keep right or toggle wrap, see my answer here https://github.com/ibhagwan/fzf-lua/discussions/1463#discussioncomment-10792276
